### PR TITLE
Include team fields in ORA XML import/export

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** include team-related settings in the export/import ORA XML.

**What changed?**

- When set, write `teams_enabled` and `selected_teamset_id` to XML when exporting ORA Xblock.
- Read the above fields from XML and use to set fields on imported ORA Xblock.
- Add unit tests for new behavior.
- Remove some test setup made redundant by the new changes.
- Bump ORA to `2.9.7`

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5281](https://openedx.atlassian.net/browse/EDUCATOR-5281)

FIY: @edx/masters-devs-gta

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='2.9.6',
+    version='2.9.7',
```
Comment (muselesscreator) [inline setup.py:39]: Will need to bump one more time.

**Comment by muselesscreator** [comment]
Haven't gotten a chance to manually test, but the changes LGTM.
You will need to bump the version again tho :-p
